cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -9,51 +9,100 @@
+    argnames, argvalues, idlist = testgen.infra_providers(metafunc, "vm_analysis_new")
 
-    new_argvalues = []
     new_idlist = []
+    new_argvalues = []
+
     for i, argvalue_tuple in enumerate(argvalues):
         args = dict(zip(argnames, argvalue_tuple))
-        if not args['provisioning']:
-            # Don't know what type of instance to provision, move on
-            continue
 
-        if not {'analysis-username', 'analysis-pass', 'analysis-image'}.issubset(
-                args['provisioning'].viewkeys()):
-            # Need all for template provisioning
+        vms = []
+        provisioning_data = []
+
+        try:
+            vms = argvalue_tuple[0].get("vms")
+            provisioning_data = argvalue_tuple[0].get("provisioning")
+        except AttributeError:
+            # Provider has no provisioning and/or vms list set
             continue
 
-        # 'analysis-template' might be omitted
-        if 'analysis-template' in args['provisioning']:
-            cloud_init_template = args['provisioning']['analysis-template']
-            if cloud_init_template not in cfme_data.get('customization_templates', {}).keys():
+        for vm_analysis_key in vms:
+            # Each VM can redefine a provisioning data
+            vm_analysis_data = provisioning_data.copy()
+            vm_analysis_data.update(vms[vm_analysis_key])
+
+            if not {'image', 'fs-type'}.issubset(
+                    vm_analysis_data.viewkeys()):
                 continue
 
-            if 'cloud_init_template' not in argnames:
-                argnames = argnames + ['cloud_init_template']
-            argvalues[i].append(get_template_from_config(cloud_init_template))
+            if vm_analysis_data['fs-type'] not in ['ntfs', 'fat32']:
+                # Username and password are required for non-windows VMs
+                if not {'username', 'password'}.issubset(
+                        vm_analysis_data.viewkeys()):
+                    continue
 
-        new_idlist.append(idlist[i])
-        new_argvalues.append(argvalues[i])
+            # Set VM name here
+            vm_name = 'test_ssa_{}-{}'.format(fauxfactory.gen_alphanumeric(), vm_analysis_key)
+            vm_analysis_data['vm_name'] = vm_name
+
+            new_idlist.append('{}-{}'.format(idlist[i], vm_analysis_key))
+            new_argvalues.append([vm_analysis_data, args["provider"]])
 
     testgen.parametrize(metafunc, argnames, new_argvalues, ids=new_idlist, scope="module")
 
 
 @pytest.fixture(scope="module")
-def local_setup_provider(request, setup_provider_modscope, provider, provisioning):
+def local_setup_provider(request, setup_provider_modscope, provider, vm_analysis_new):
+    if provider.type == 'rhevm' and version.current_version() < "5.5":
+        # See https://bugzilla.redhat.com/show_bug.cgi?id=1300030
+        pytest.skip("SSA is not supported on RHEVM for appliances earlier than 5.5 and upstream")
```

Comment:
This maybe?

```python
# At the top of the file
pytestmark = [pytest.mark.uncollectif(lambda provider: provider.type == 'rhevm' and version.current_version() < "5.5")]
```

So it will apply on all tests.